Transfer for compatitive dialogue

ivanromakh opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/openprocurement/openprocurement.relocation.api/11)
<!-- Reviewable:end -->

ivanromakh commented:
Для того щоб протестувалося треба змержити openua  a105191787427142_transfer_token з production

vmaksymiv commented on openprocurement/relocation/api/tests/tender.py:
```diff
@@ -186,6 +188,30 @@ def test_change_tender_ownership(self):
+    invalid_owner = 'broker1'
+
+    def test_change_tender_ownership(self):
+        super(OpenUACompatitiveTenderOwnershipChangeTest, self).test_change_tender_ownership()
```
Що цей рядок робить?

vmaksymiv commented on openprocurement/relocation/api/tests/tender.py:
```diff
@@ -186,6 +188,30 @@ def test_change_tender_ownership(self):
+        super(OpenUACompatitiveTenderOwnershipChangeTest, self).test_change_tender_ownership()
+
+
+class OpenEUCompatitiveTenderOwnershipChangeTest(OpenUAOwnershipWebTest, TenderOwnershipChangeTest):
```
Для Open**EU**CompatitiveTender повністю підходить базовий клас Open**UA**OwnershipWebTest?

vmaksymiv commented on buildout.cfg:
```diff
@@ -20,6 +21,7 @@ openprocurement.tender.openuadefense = git https://github.com/openprocurement/op
 openprocurement.tender.openeu = git https://github.com/openprocurement/openprocurement.tender.openeu.git branch=a105191787427142_transfer_token
 openprocurement.tender.limited = git https://github.com/openprocurement/openprocurement.tender.limited.git branch=a105191787427142_transfer_token
 openprocurement.contracting.api = git https://github.com/openprocurement/openprocurement.contracting.api.git branch=a105191787427142_transfer_token
+openprocurement.tender.competitivedialogue = git https://github.com/openprocurement/openprocurement.tender.competitivedialogue.git branch=production
```
Зміни в коді openprocurement.tender.competitivedialogue передбачаються?

vmaksymiv reviewed (commented):
В конкурентному діалозі є два етапи і для обох потрібна можливість "переїзду".

ivanromakh commented:
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

_[openprocurement/relocation/api/tests/tender.py, line 200 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWbkCNqWB64KzMU_dqn-r1-200:-KWcDUHicQmrp5t0OAXM:b9beiks) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/3297a815f1f5b52bbf7ab79fe2f04179f7b0b862/openprocurement/relocation/api/tests/tender.py#L200)):_
<details><summary>

<i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary>



> Що цей рядок робить?
> </details>
> Він просто перевизначає функцію. (можливо щоб показати, що там є тільки один тест). Воно було у всіх прикладах тому вирішив не міняти стиль 

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11)_

<!-- Sent from Reviewable.io -->

ivanromakh commented:
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

_[openprocurement/relocation/api/tests/tender.py, line 200 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWbkCNqWB64KzMU_dqn-r1-200:-KWcEed36-tcLIgY9TQw:brcar0x) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/3297a815f1f5b52bbf7ab79fe2f04179f7b0b862/openprocurement/relocation/api/tests/tender.py#L200)):_
<details><summary>

<i>Previously, ivanromakh wrote…</i></summary>



> Він просто перевизначає функцію. (можливо щоб показати, що там є тільки один тест). Воно було у всіх прикладах тому вирішив не міняти стиль 
> </details>
> Можливо той хто писав надіявся, що будуть ще щось дописувати в дану функцію тести для конкретних процедур. Так для другої стадії придеться напевно

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11)_

<!-- Sent from Reviewable.io -->

vmaksymiv commented:
Вже бачу, що таке було. Не бачу потреби в тих рядках.

ivanromakh commented:
Review status: 0 of 5 files reviewed at latest revision, 3 unresolved discussions.

---

_[buildout.cfg, line 24 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWbkCNqWB64KzMU_dql-r1-24:-KWgWZ3aTENI9yGfzMI8:barrslc) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/3297a815f1f5b52bbf7ab79fe2f04179f7b0b862/buildout.cfg#L24)):_
<details><summary>

<i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary>



> Зміни в коді openprocurement.tender.competitivedialogue передбачаються?
> </details>
> Скоріш за все ні (передачити не можу бо не працював з даним проектом)

---

_[openprocurement/relocation/api/tests/tender.py, line 200 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWbkCNqWB64KzMU_dqn-r1-200:-KWcGwUKNGfCbcTknyg5:b9otfed) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/3297a815f1f5b52bbf7ab79fe2f04179f7b0b862/openprocurement/relocation/api/tests/tender.py#L200)):_
<details><summary>

<i>Previously, ivanromakh wrote…</i></summary>



> Можливо той хто писав надіявся, що будуть ще щось дописувати в дану функцію тести для конкретних процедур. Так для другої стадії придеться напевно
> </details>
> Для другої стадії прийдеться імпортнути клас який власне створює тендери для 2-гої стадії

---

_[openprocurement/relocation/api/tests/tender.py, line 203 at r1](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWbkCNqWB64KzMU_dqn-r1-203:-KWgWki4yznDYi80u3Lm:b9wjs8b) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/3297a815f1f5b52bbf7ab79fe2f04179f7b0b862/openprocurement/relocation/api/tests/tender.py#L203)):_
<details><summary>

<i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary>



> Для Open**EU**CompatitiveTender повністю підходить базовий клас Open**UA**OwnershipWebTest?
> </details>
> Змінив

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11)_

<!-- Sent from Reviewable.io -->

vmaksymiv commented on .travis.yml:
```diff
@@ -1,5 +1,9 @@
-sudo: false
+sudo: required
+before_script:
+  - sudo add-apt-repository ppa:chris-lea/libsodium -y
```
цих змін додавати не треба. openprocurement.relocation.api не має залежностей на ті бібліотеки.

ivanromakh commented:
Review status: 0 of 9 files reviewed at latest revision, 4 unresolved discussions.

---

_[.travis.yml, line 4 at r2](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWhj5n7BukFYWm9PoW3-r2-4:-KWhvBlE6SJG6GVIVrtd:b-9h0uyf) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/e37b7198cf857f3ba5db0cbd5e661a1c584f50a0/.travis.yml#L4)):_
<details><summary>

<i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary>



> цих змін додавати не треба. openprocurement.relocation.api не має залежностей на ті бібліотеки.
> </details>
> Не працює без libsodium на жаль

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11)_

<!-- Sent from Reviewable.io -->

vmaksymiv commented:
Review status: 0 of 9 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

_[.travis.yml, line 4 at r2](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWhj5n7BukFYWm9PoW3-r2-4:-KWiGVm0T9mSqtktgJ1H:b7mknon) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/e37b7198cf857f3ba5db0cbd5e661a1c584f50a0/.travis.yml#L4)):_
<details><summary>

<i>Previously, ivanromakh wrote…</i></summary>



> Не працює без libsodium на жаль
> </details>
> Таки так. Зміни потрібні. Перепрошую.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11)_

<!-- Sent from Reviewable.io -->

ivanromakh commented:
Сплющив останні коміти

ivanromakh committed:
Add compatitive_dialogue to relocation

ivanromakh committed:
import teder_data from competitive_dialogue

ivanromakh committed:
Add tests for bids

Add test for complaints

Test for tender stage1 and stage2

update test for tender stage2

Base class for test tender stage2

ivanromakh committed:
Update validation for ownerchange

ivanromakh committed:
Use my repository

ivanromakh commented:
Review status: 0 of 8 files reviewed at latest revision, 4 unresolved discussions.

---

*[.travis.yml, line 4 at r2](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11#-KWhj5n7BukFYWm9PoW3-r2-4:-K_93-uzQEj3Fck_l4hy:b33kw1) ([raw file](https://github.com/openprocurement/openprocurement.relocation.api/blob/e37b7198cf857f3ba5db0cbd5e661a1c584f50a0/.travis.yml#L4)):*
<details><summary><i>Previously, vmaksymiv (Volodymyr Maksymiv) wrote…</i></summary><blockquote>

Таки так. Зміни потрібні. Перепрошую.
</blockquote></details>

ок

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.relocation.api/11)*
<!-- Sent from Reviewable.io -->

ivanromakh committed:
update test for tender stage2

ivanromakh committed:
Add tender ownerchange (stage2) in docs
